### PR TITLE
Optimise/refine `radio`[`-tower`]/add `airdrop`/Apple/mac/iOS/app related icons

### PR DESCRIPTION
**Pull request by danielbayley**
`airdrop` is missing, considering we already have https://lucide.dev/icons/airplay.

`dock` (also desktop) and `app-window-mac` are for desktop apps etc, particularly for use in preference panes, for features relating to the dock/desktop/menu bar etc…

### CODE REVIEW

File: icons/radio-tower.svg [hunk 1]
```diff
@@ -9,11 +9,11 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M4.9 16.1C1 12.2 1 5.8 4.9 1.9" />
-  <path d="M7.8 4.7a6.14 6.14 0 0 0-.8 7.5" />
+  <path d="M5 2a10 10 0 0 0 .7 14.8" />
+  <path d="M7.8 4.8c-2.1 2.1-2.3 5.4-.6 7.8" />
```
Comment (karsa-mistmere) [inline icons/radio-tower.svg:13]:
```suggestion
  <path d="M4.93 16.07A10 10 0 0 1 4.86 2" />
  <path d="M7.76 4.76a6 6 0 0 0-.59 7.8" />
```

File: icons/podcast.svg [hunk 2]
```diff
@@ -9,8 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <circle cx="12" cy="11" r="1" />
-  <path d="M11 17a1 1 0 0 1 2 0c0 .5-.34 3-.5 4.5a.5.5 0 0 1-1 0c-.16-1.5-.5-4-.5-4.5Z" />
-  <path d="M8 14a5 5 0 1 1 8 0" />
   <path d="M17 18.5a9 9 0 1 0-10 0" />
```
Comment (karsa-mistmere) [inline icons/podcast.svg:12]:
```suggestion
  <path d="M16.85 18.58a9 9 0 1 0-9.7 0" />
```

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

**Comment by karsa-mistmere** [comment]
AirDrop and AirPlay are Apple trademarks, not only will AirDrop not be added, AirPlay will also be removed.

**Comment by ericfennis** [comment]
Maybe not a bad idea @danielbayley

File: icons/radio-tower.svg [hunk 3]
```diff
@@ -9,11 +9,11 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M4.9 16.1C1 12.2 1 5.8 4.9 1.9" />
-  <path d="M7.8 4.7a6.14 6.14 0 0 0-.8 7.5" />
+  <path d="M4.93 16.07A10 10 0 0 1 4.86 2" />
```
Comment (ericfennis) [inline icons/radio-tower.svg:12]: This one is adding more code right?

**Pull request merged**